pages/index.js

```diff
@@ -18,36 +21,129 @@ import {
   LatestBlockTXNIcon,
   SecondsToBlockIcon,
   TotalSupplyIcon,
-} from 'svgx'
+} from 'svgx';
 
+const LastBlockInfo = () => {
+  const cardWidth = useBreakpointValue({
+    base: '100%',
+    sm: 'calc(50% - 1rem)',
+    md: 'calc(33% - 1rem)',
+  });
+  const headBlock = useBlockHead();
```

Comment:
So this might be too late relative to how we are doing things today -- I have a convention in the previous codebases where any value which comes from a `use` hook is prefixed with a `$` so that you can know at a glance that this value is dynamic.

How would you feel about migrating to that convention in the future?

Reply:
Nice idea. I tryed to replace it. Please check.